core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/ListTasksRequest.java

```diff
@@ -47,15 +48,31 @@ public ListTasksRequest detailed(boolean detailed) {
         return this;
     }
 
+    /**
+     * Should this request wait for all found tasks to complete?
+     */
+    public boolean shouldWaitForCompletion() {
+        return waitForCompletion;
+    }
+
+    /**
+     * Should this request wait for all found tasks to complete?
+     */
+    public void setWaitForCompletion(boolean waitForCompletion) {
```

Comment:
This is a Request and setters in requests don't typically have "set" prefix and return the request itself. I don't like this notation, but this is the notation that is used in most of the existing requests including this one. Check detailed setting above.